app/models/event.rb

```diff
@@ -102,6 +102,14 @@ def area_title
                     }
                   },
                   associated_against: { area: :name }
+  multisearchable against:
+                    [:non_billig_title_no,
+                     :title_en,
+                     :long_description_en,
+                     :long_description_no,
+                     :age_limit,
```

Comment:
could add type, but would have to add both the norwegian and english translations, will have to look into that. I also considered adding organizer